http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -386,10 +389,7 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift](
             }
           } yield ()
         } else {
-          for {
-            _ <- runtimeConfig.cloudService.interpreter.updateMachineType(UpdateMachineTypeParams(runtime, m, now))
-            _ <- RuntimeConfigQueries.updateMachineType(runtime.runtimeConfigId, m, now).transaction
```

Comment:
I don't think we needed to update the DB here because `runtimeConfig.cloudService.interpreter.updateMachineType` already does that